rele/contrib/logging_middleware.py

```diff
@@ -48,6 +48,20 @@ def pre_publish(self, topic, data, attrs):
             },
         )
 
+    def post_publish_failure(self, topic, exception, message):
+        self._logger.exception(
+            f"Exception raised while processing message "
```

Comment:
What do you think to relate the log content to the "publish" failure event?

```suggestion
            f"Exception raised while publishing message "
```